moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -33,17 +33,19 @@
 
   <class name="default_planner_request_adapters/AddTimeParameterization" type="default_planner_request_adapters::AddTimeParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
     <description>
+      A deprecated time parameterization plugin.
```

Comment:
So does this mean TOTG is the only algorithm recommended to use? Maybe this information should be added to the tutorials as well.

Reply:
Yep, I agree.

Reply:
I don't think anyone will ever notice this description. Could you add some warnings if the old plugins are being used? I'm also not convinced the old plugins should be deprecated, switching the default to TOTG and hinting on the shortcomings seems sufficient to me.

Reply:
In the interest of getting this merged, I'll just revert the changes to `TimeParameterization` and `IterativeSplineParameterization`